### DISCUSSION TITLE
Pressure not supported?

### DISCUSSION THREAD

**Discussion by gerroon** [flxzt/rnote#1052] [Q&A]
HI
I have a Wacom tablet and using this on Windows 11. I do not see to have any kind of pressure sensitivity with the pen or other tool. Is this supported?

**Comment by Doublonmousse** [comment]
Yes, pressure is supported in the app.

Do make sure you use settings that enable pressure (although by default it's enabled).

Rnote uses gtk4 that itself uses Windows Ink for pen inputs. Maybe you can check in the Wacom settings panel if you can enable Windows Ink there (activating it might make some configuration unavailable in the process, if I remember correctly but it's enough to test if that's the issue or not).

I'm pretty sure that not using the Windows Ink make rnote treat pen input as mouse inputs, without an associated pressure.

I could also test this myself, I have a wacom tablet lying around, just need to install the drivers to test this.

**Reply by gerroon** [reply]
I had to disable and enable the windows ink again and it works now. Thanks for the help.

**Reply by Doublonmousse** [reply]
Note :  for wacom tablet in particular, windows ink disable some features of the wacom driver that you would otherwise have (there's no relative mode for example).

You can try open tablet driver + install the Windows Ink plugin if that bothers you, and I can confirm I can get it to work with pressure support.

**Reply by gerroon** [reply]
Thanks for the info. Where do I install the Windows Ink plugin?

**Reply by Doublonmousse** [reply]
You have to install open tablet driver and there's some additional step for the windows ink plugin (see here https://github.com/X9VoiD/VoiDPlugins/wiki/WindowsInk )
